src/pattern-library/components/PlaygroundApp.tsx

```diff
@@ -19,130 +91,125 @@ export default function PlaygroundApp({
-    activeRoute && activeRoute.component ? (
-      <activeRoute.component />
-    ) : (
-      <Library.Page title=":( Sorry">
-        <h1 className="text-2xl">Page not found</h1>
-      </Library.Page>
-    );
-
-  /**
-   * Header for a primary section of nav
-   */
-  function NavHeader({ children }: { children: ComponentChildren }) {
-    return (
-      <h2 className="border-b px-2 py-1 font-light text-lg">{children}</h2>
-    );
-  }
 
-  /**
-   * A single navigation link
-   */
-  function NavLink({ route }: { route: PlaygroundRoute }) {
-    return (
-      <li className="-ml-[2px]">
-        {route.route && (
-          <Link
-            classes={classnames('pl-4 w-full border-l-2 hover:border-l-brand', {
-              'border-l-2 border-brand font-semibold':
-                activeRoute?.route === route.route,
-              'border-transparent': activeRoute?.route !== route.route,
-            })}
-            href={`${baseURL}${route.route.toString()}`}
-          >
-            {route.title}
-          </Link>
-        )}
-        {!route.route && (
-          <div className="pl-4 w-full text-slate-5 font-light">
-            {route.title}
-          </div>
-        )}
-      </li>
-    );
-  }
-
-  /**
-   * Render a list of navigation items
-   */
-  function NavList({ routes }: { routes: PlaygroundRoute[] }) {
-    return (
-      <ul className="ml-2 space-y-2 border-l-2 border-slate-0">
-        {routes.map(route => (
-          <NavLink key={route.title} route={route} />
+  const pageRoutes = (
+    <>
+      {allRoutes
+        .filter(route => !!route.route)
+        .map(aRoute => (
+          <Route key={aRoute.title} path={aRoute.route}>
+            {aRoute.component ?? aRoute.title}
+          </Route>
         ))}
-      </ul>
-    );
-  }
+    </>
+  );
 
-  /**
-   * Secondary section of navigation, with header
-   */
-  function NavSection({
-    title,
-    children,
-  }: {
-    title: string;
-    children: ComponentChildren;
-  }) {
-    return (
-      <div className="space-y-4">
-        <h3 className="mx-2 text-slate-7 font-semibold text-sm">{title}</h3>
-        {children}
-      </div>
-    );
-  }
   const groupKeys = Object.keys(componentGroups) as Array<
     keyof typeof componentGroups
   >;
   return (
-    <main className="max-w-[1200px] mx-auto">
-      <div className="md:grid md:grid-cols-[2fr_5fr]">
-        <div className="md:h-screen md:sticky md:top-0 overflow-scroll">
-          <div className="md:sticky md:top-0 h-16 px-6 flex items-center bg-slate-0 border-b">
-            <h1 className="text-lg">
-              <Link href={baseURL + '/'} classes="grow flex gap-x-2">
-                <LogoIcon />
-                Component Library
-              </Link>
-            </h1>
-          </div>
-          <nav id="nav" className="pt-8 pb-16 space-y-4 mr-4">
-            <NavHeader>Foundations</NavHeader>
-            <NavList routes={getRoutes('foundations')} />
+    <Router base={baseURL}>
```

Comment:
NB: Except for wrapping the whole thing in a `Router`, there are no `wouter`-specific changes here.